go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -87,25 +96,29 @@ func NewEmergencyReparenter(ts *topo.Server, tmc tmclient.TabletManagerClient, l
 // ReparentShard performs the EmergencyReparentShard operation on the given
 // keyspace and shard.
 func (erp *EmergencyReparenter) ReparentShard(ctx context.Context, keyspace string, shard string, opts EmergencyReparentOptions) (*events.Reparent, error) {
+	// First step is to lock the shard for the given operation
 	opts.lockAction = erp.getLockAction(opts.NewPrimaryAlias)
-
 	ctx, unlock, err := erp.ts.LockShard(ctx, keyspace, shard, opts.lockAction)
 	if err != nil {
 		return nil, err
 	}
-
+	// defer the unlock-shard function
```

Comment:
I'm not sure this comment is adding any value that the variable name + `defer` doesn't already provide

Reply:
Removed